lib/remote/jsonrpcconnection.cpp

```diff
@@ -245,9 +274,15 @@ void JsonRpcConnection::Disconnect()
 	});
 }
 
-void JsonRpcConnection::MessageHandler(const String& jsonString)
+void JsonRpcConnection::MessageHandler(const String& jsonString, String& rpcMethod)
```

Comment:
The purpose of the new parameter is not obvious from the function signature. I'm not a huge fan of doing it that was, I haven't thought about alternative ways to get that information where it's needed, but at the very least, this would need documentation.